assets/html/js/metadata.js

```diff
@@ -0,0 +1,8 @@
+// libraries: jquery
+// arguments: $
+
+let meta = $("div[data-docstringscollapsed]").data();
+
+if (meta.docstringscollapsed) {
+  $("#documenter-article-toggle-button").click();
```

Comment:
This seems fine to me. It still shows the close-animation after the page is loaded. I don't mind it that much, but @mortenpi might still prefer if we speed it up.

@Hetarth02 Do you know what to add to control the animation speed?